### ISSUE TITLE
No Training with customised dataset for single class detection

### ISSUE THREAD

**Issue by cmrs2017** [fizyr/keras-retinanet#682]
Training starts and continuous with no error. But the regression loss (2.18) /acc(0.25) and classification loss(2.3)/acc (.981) is not improving. I tried tuning the learning rates and make all layers trainable (including batch normalisation layers) but no luck. 

My problem is a single class detection.
Experimental setting:  Backbone= resnet-50, Image large side size max - 480, Image short side size min - 380 
System Configuration : tensorflow-gpu-1.4.0 , Keras 2.2.0 , Cuda 8

Can anyone suggest on how I can train with customized dataset using retinanet ? 
Is retinanet a wrong choice for single class detection problem ?

**Comment by hgaiser** [comment]
Tensorflow 1.4.0 is quite old, could you try updating to the latest tensorflow and try again?

**Comment by PhilippMarquardt** [comment]
I trained one for single class detection yesterday. It worked fine with no problems. So there is no problem with single class detection in the library.

**Comment by hgaiser** [comment]
Closing due to inactivity (and assuming your tensorflow is too old).

**Comment by cmrs2017** [comment]
The network is training with tensor flow 1.4.0. I just updated the TF operations. Thank you for the suggestion.
